model/src/main/kotlin/utils/Extensions.kt

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
commit-message: As this commit goes a bit in the oposite direction of "making all the finding paths relative to analyzer root", which I still think as a chance to turn out as preferred direction - shall we mention this in the commit message?

Reply:
Done.

Reply:
Thanks!